src/tsv.js

```diff
@@ -72,8 +74,28 @@ define([
         return rows;
     }
 
+    function escape(value) {
+        if (value === null || _.isUndefined(value)) {
```

Comment:
Should also test for value === false. Or do `if (!value)` - this seems like a case where writing non-strict code actually does the right thing.